spec/index.bs

```diff
@@ -3292,6 +3292,7 @@ Issue(https://github.com/gpuweb/gpuweb/issues/693): add a limit to the number of
 
 <script type=idl>
 interface GPUCommandBuffer {
+    double getExecutionTime();
```

Comment:
This would need to be a promise, because we won't know the execution time until the GPU is finished executing the command buffer. And also it can be a plain member because Javascript engines can transform member accesses in function calls.

Reply:
```suggestion
    readonly attribute Promise<double> executionTime;
```